pkg/config/consts.go

```diff
@@ -40,7 +38,7 @@ const (
 
 	// Consensus-related settings
 	// Protocol-based consensus step time.
-	ConsensusTimeOut = 5 * time.Second
+	DefaultConsensusTimeOutSeconds = 5
```

Comment:
Pure curiosity: Can we have the const value be a `time.Duration`? Something that may be worth considering.